tezos/new_context/src/tezedge_context.rs

```diff
@@ -297,28 +304,31 @@ impl TezedgeIndex {
         };
 
         // first get node at key
-        let child_node_id = match storage.get_tree_node_id(root, first) {
-            Some(hash) => hash,
+        let child_node_id = match storage.tree_find_node(root, first) {
+            Some(node_id) => node_id,
             None => {
                 return Ok(Tree::empty());
```

Comment:
For later (cleanup), but this `find_raw_tree` should probably return an `Option`